tests/test_databag.py

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+import os
+
+from lektor.builder import Builder
+from lektor.db import Database
+from lektor.environment import Environment
+from lektor.project import Project
+
+
```

Comment:
+1 for the tests, but I would suggest these tests be converted to unit tests.

Test the data loading functionality with tests that call `lektor.databags.load_databag` directly.

If feeling ambitious, test the methods of the `Databags` class by constructing a `Databags` instance and calling the methods directly.

This makes it easier to check for specific issues (e.g., is the parsing to OrderedDicts working correctly?) It allows for more tests (to check various aspects of loading) while keeping total test run time low.

If you want to include an integration test to ensure that the `bag()` works in a template, that's okay.  My personal feeling is that is the lowest priority of the tests — if that's broken, someone is likely to notice, even without the test in place.